iothub/service/tests/IotHubSasCredentialPropertiesTests.cs

```diff
@@ -116,5 +116,21 @@ public async Task TestCbsTokenGeneration_MissingExpiration_Fails()
                 ex.Message.Should().Be($"There is no expiration time on {nameof(AzureSasCredential)} signature.");
```

Comment:
I know this test wasn't added in this PR, but is there a better way for us to test this? Comparing error message texts isn't the most reliable way to test this.